### DISCUSSION TITLE
Welcome to SFGUI Discussions!

### DISCUSSION THREAD

**Discussion by TankOs** [TankOs/SFGUI#87] [General]
# 👋 Welcome!

Feel free to ask your SFGUI-related questions here. Have fun. :-)

**Comment by BDOTimer** [comment]
collected SFGUI.dll weighing 11 mb and examples. Running an example Box.exe and as a result, I get a window with a log:
![1](https://github.com/user-attachments/assets/92de135d-4415-4e2f-b74d-6214e07ba562)

How to fix it?

**Reply by eXpl0it3r** [reply]
Would be better to open a separate discussion thread.

Do you have a dedicated GPU? Are you running it in a VM? What version of SFGUI are you using?

**Reply by BDOTimer** [reply]
> Would be better to open a separate discussion thread.

I am not the moderator of this discussion - you should know better.

> Do you have a dedicated GPU? Are you running it in a VM? What version of SFGUI are you using?
1.
GPU nvidia 1050 Ti.

2.
I am not interested in VM.

3.
https://github.com/TankOs/SFGUI
cmake for mingw32-make.exe

4.
To be very brief, I want to connect sfgui to SFML 3.0, all the examples of which work fine for me.
